internal/framework/service/gateway_app_types/data_source_test.go

```diff
@@ -0,0 +1,40 @@
+package gateway_app_types_test
+
+import (
+	"fmt"
+	"os"
+	"testing"
+
+	"github.com/cloudflare/terraform-provider-cloudflare/internal/acctest"
+	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+)
+
+func TestAccCloudflareGatewayAppTypes_DataSource(t *testing.T) {
+	accountID := os.Getenv("CLOUDFLARE_ACCOUNT_ID")
+
+	if accountID == "" {
+		t.Fatal("CLOUDFLARE_ACCOUNT_ID must be set for acceptance tests")
```

Comment:
this should be in https://github.com/cloudflare/terraform-provider-cloudflare/blob/5d4c93cdfc5387dbb8603b94808c8b943a6f8aef/internal/acctest/acctest.go#L64 however, we haven't had anything rely on it yet so let's move it in there now and update the `PreCheck` in the test itself.